core/src/main/java/com/linecorp/armeria/server/logging/ContentPreviewingService.java

```diff
@@ -147,10 +147,11 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
             return unwrap().serve(ctx, req);
         }
         ctx.setAttr(SETTING_CONTENT_PREVIEW, true);
-        final ContentPreviewer requestContentPreviewer =
-                contentPreviewerFactory.requestContentPreviewer(ctx, req.headers());
-        req = setUpRequestContentPreviewer(ctx, req, requestContentPreviewer, requestPreviewSanitizer);
-
+        if (!req.isEmpty()) {
```

Comment:
I haven't been able to check this in detail but does this change mean:
- previously: `requestContentPreview == ""` (empty string)
- after: `requestContentPreview` == null`

I'm slightly concerned users won't be able to differentiate between 1) content preview not showing 2) content preview is actually empty

Reply:
That's a good point. I set the empty String explictly.